environment_test.py

```diff
@@ -0,0 +1,7 @@
+import dotenv
+import os
+
+
+dotenv. load_dotenv()
+
+print(os.getenv("TRELLO_API_KEY"))
```

Comment:
These sorts of scripts are really useful for testing, but hopefully unnecessary once we've got them working, so a good idea to delete this now to prevent it clogging up the codebase